docs/user_manual/source/instruction_fetch.rst

```diff
@@ -27,23 +27,25 @@ are possible and thus it needs fewer signals.
+  | ``instr_gnt_i``            | input           | The other side accepted the request. ``instr_addr_o``, ``instr_mem_type_o`` and ``instr_prot_o`` may change in the next cycle.                    |
+  +----------------------------+-----------------+---------------------------------------------------------------------------------------------------------------------------------------------------+
+  | ``instr_addr_o[31:0]``     | output          | Address, word aligned                                                                                                                             |
+  +----------------------------+-----------------+---------------------------------------------------------------------------------------------------------------------------------------------------+
+  | ``instr_mem_type_o[1:0]``  | output          | Memory Type attributes (cacheable, bufferable)                                                                                                    |
+  +----------------------------+-----------------+---------------------------------------------------------------------------------------------------------------------------------------------------+
+  | ``instr_prot_o[2:0]``      | output          | Protection attributes                                                                                                                             |
+  +----------------------------+-----------------+---------------------------------------------------------------------------------------------------------------------------------------------------+
+  | ``instr_rvalid_i``         | input           | ``instr_rdata_i`` and ``instr_err_i`` are valid when ``instr_rvalid_i`` is high. This signal will be high for exactly one cycle per request.      |
+  +----------------------------+-----------------+---------------------------------------------------------------------------------------------------------------------------------------------------+
+  | ``instr_rdata_i[31:0]``    | input           | Data read from memory                                                                                                                             |
+  +----------------------------+-----------------+---------------------------------------------------------------------------------------------------------------------------------------------------+
+  | ``instr_err_i``            | input           | An instruction interface error occurred                                                                                                           |
+  +----------------------------+-----------------+---------------------------------------------------------------------------------------------------------------------------------------------------+
 
```

Comment:
Missing *achk / *rchk ?

Reply:
I know I left these out (also didn't upload the new OBI spec yet); will add a preliminary description.